.travis.yml

```diff
@@ -23,4 +23,5 @@ jobs:
     - stage: Dependency Manager Validation
       name: CocoaPods
       script:
+      - travis_wait 20
```

Comment:
@marcpowell-okta Could you please review this change? Previous builds were failing due to timeout error, so I increased timeout here from 10 to 20 minutes.